applications/MeshMovingApplication/tests/test_results/mesh_moving_rectangle_structural_similarity_2D4N_results.json

```diff
@@ -0,0 +1 @@
+{"TIME": [0.2, 0.30000000000000004, 0.5], "NODE_55": {"MESH_DISPLACEMENT_X": [0.5996296467199931, -217.19207868149792, -29033349.487693418], "MESH_DISPLACEMENT_Y": [0.08072963054358105, -29.232764059679344, -3907724.5270597623], "MESH_DISPLACEMENT_Z": [0.0, 0.0, 0.0], "MESH_VELOCITY_X": [0.06595926113919924, -11.85262173224079, -1588914.8072574805], "MESH_VELOCITY_Y": [0.008880259359793917, -1.5952889305481095, -213858.9405877449], "MESH_VELOCITY_Z": [0.0, 0.0, 0.0]}, "NODE_95": {"MESH_DISPLACEMENT_X": [4.5251559830210475, -1638.0601369730375, -218970152.59532025], "MESH_DISPLACEMENT_Y": [1.5204292042307508, -552.6019807582369, -73867746.36491822], "MESH_DISPLACEMENT_Z": [0.0, 0.0, 0.0], "MESH_VELOCITY_X": [0.4977671581323153, -89.39190835614882, -11983629.993660953], "MESH_VELOCITY_Y": [0.1672472124653826, -30.15744241504727, -4042577.180357512], "MESH_VELOCITY_Z": [0.0, 0.0, 0.0]}}
```

Comment:
referencing for #4324 
@roigcarlo here the node-ids are used for checking the values